### ISSUE TITLE
Any way to disable inline styles?

### ISSUE THREAD

**Issue by LittleHamster** [siteorigin/siteorigin-panels#363]
Is there any way that we can disable the inline css styles? for example when adding a background color, this is added to the element: 

```
<div class="some-class" style="backgroud-color:#000;" > content </div>
```

can this be view a css file? there is numerous reasons why inline style is bad. 

> Don't inline CSS attributes
> Inlining CSS attributes on HTML elements (e.g., p style=...) should be avoided where possible, as this often leads to unnecessary code duplication. Further, inline CSS on HTML elements is blocked by default with Content Security Policy (CSP). [[source]](https://developers.google.com/speed/docs/insights/OptimizeCSSDelivery)

imagine if someone is customizing a lot of elements and having all these inline styles :(

**Comment by gregpriday** [comment]
This isn't possible at the moment, but we do have an open issue for moving all styles into the generated CSS https://github.com/siteorigin/siteorigin-panels/issues/41 - you can keep track of the progress there.

I'll take a look at this today.

**Comment by gregpriday** [comment]
I've managed to resolve #41. This should be merged in time for Page Builder 2.5
